src/helpers/domains.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from 'ethers';
+
+const getSubnodeHash = (parentHash: string, labelHash: string): string => {
+	const calculatedHash = ethers.utils.keccak256(
+		ethers.utils.defaultAbiCoder.encode(
+			['bytes32', 'bytes32'],
+			[ethers.utils.arrayify(parentHash), ethers.utils.arrayify(labelHash)],
+		),
+	);
+
+	return calculatedHash;
+};
```

Comment:
If `getSubnodeHash` is only used in `getDomainId`, it might be cleaner to just move the implementation into `getDomainId`.

Reply:
removed this function, we never use this!!